### PR TITLE
Contacts page

### PR DESCRIPTION
**Pull request by gunbux**
### Changes made
- Added contents page
- Modified ClickToCopySection

2022-01-13 19 51 39 localhost 9351705878b9

### CODE REVIEW

**Comment by gunbux** [comment]
[contacts page](https://trello.com/c/5L55A75o/20-contacts-page)

File: src/components/ClickToCopySection/styles/ClickToCopySection.styled.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+  font-weight: 300;
+  font-size: 1.25vw;
   line-height: 50px;
 `
 
 export const ClickToCopyLink = styled.a`
   font-style: normal;
   font-weight: 300;
-  font-size: 30px;
+  font-size: 2vw;
```
Comment (hui444) [inline src/components/ClickToCopySection/styles/ClickToCopySection.styled.ts:15]: this too, consider using clamp?

File: src/components/ClickToCopySection/styles/ClickToCopySection.styled.ts [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 export const ClickToCopyTitle = styled.p`
-  margin: 0px;
+  margin: 0;
 
   font-style: normal;
-  font-weight: 500;
-  font-size: 18px;
+  font-weight: 300;
+  font-size: 1.25vw;
```
Comment (hui444) [inline src/components/ClickToCopySection/styles/ClickToCopySection.styled.ts:8]: would this make text too big if window width is too large? consider using clamp instead

File: src/pages/Contact/index.tsx [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+                  {/* @ts-ignore */}
+                  <ClickToCopySection text={contactMethods[header]} title={header} />
+                </SectionContainer>
+              )
+            }
+            return (
+              <SectionContainer>
+                <SectionTitle>{header}</SectionTitle>
+                <SectionContent>
+                  {/* TODO: Link to actual socials? */}
```
Comment (hui444) [inline src/pages/Contact/index.tsx:38]: consider updating rhdevsInfo to include the links to use them, and remove 'facebook'

File: src/pages/Contact/styles/Contact.styled.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+export const SectionContainer = styled.div`
+  margin: 30px 0;
+`
+
+export const SectionTitle = styled.p`
+  margin: 0;
+
+  font-style: normal;
+  font-weight: 300;
+  font-size: 1.25vw;
```
Comment (hui444) [inline src/pages/Contact/styles/Contact.styled.ts:35]: consider using clamp?

File: src/pages/Contact/styles/Contact.styled.ts [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  line-height: 50px;
```
Comment (hui444) [inline src/pages/Contact/styles/Contact.styled.ts:36]: any reason why this is at 50px? consider using values like 1 or 1.2 etc?

File: src/pages/Contact/styles/Contact.styled.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+export const SectionContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+export const ContactLink = styled.div`
+  font-style: normal;
+  font-weight: 300;
+  font-size: 2vw;
```
Comment (hui444) [inline src/pages/Contact/styles/Contact.styled.ts:48]: consider using clamp?

File: src/pages/Contact/styles/Contact.styled.ts (hunk 5 above)
Comment (gunbux) [reply src/pages/Contact/styles/Contact.styled.ts:36]: i just took the values off the figma, but will probably swap to relative values

**Review by hui444 — APPROVED** [review]
lgtm but consider using clamp or theme's typography for text stylings (if needed, can add text styles)

File: src/pages/Contact/styles/Contact.styled.ts [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+export const TextSection = styled.div`
+  width: 50%;
+  margin-right: 50px;
+`
+
+export const ContactTitle = styled.div<{ fontType: FontType }>`
+  ${fontTypeCss}
+  //font-weight: 300;
+  //font-size: 4vw;
```
Comment (hui444) [inline src/pages/Contact/styles/Contact.styled.ts:24]: remove if not in use!

**Pull request merged**